src/redis-cli.c

```diff
@@ -8210,6 +8222,18 @@ static void intrinsicLatencyModeStop(int s) {
     force_cancel_loop = 1;
 }
 
+static void sigIntHandler(int s) {
+    UNUSED(s);
+
+    if (config.monitor_mode || config.pubsub_mode) {
```

Comment:
if we remove the if, look like we can abort from BLOCKING command, like BPOP key 0
https://github.com/antirez/linenoise/blob/master/linenoise.c#L854
```
[root@binblog redis]# src/redis-cli
127.0.0.1:6379> blpop key 0
^C(1.69s)
127.0.0.1:6379> ping
PONG
```

also the else part, it look like it won't be able to touch.